sdks/python/apache_beam/examples/snippets/transforms/element_wise/pardo.py

```diff
@@ -47,16 +49,16 @@ def process(self, text):
 
 
 def pardo_dofn_params(test=None):
+  # pylint: disable=line-too-long
   # [START pardo_dofn_params]
   import apache_beam as beam
 
-  # pylint: disable=line-too-long
   class AnalyzeElement(beam.DoFn):
     def process(self, elem, timestamp=beam.DoFn.TimestampParam, window=beam.DoFn.WindowParam):
       yield '\n'.join([
           '# timestamp',
           'type(timestamp) -> ' + repr(type(timestamp)),
-          'timestamp.micros -> ' + repr(timestamp.micros),
+          'timestamp.micros -> ' + repr(int(timestamp.micros)),
```

Comment:
I think this may cause questions for readers of this code.
Given that Python 2 is going away, how about we just skip the test if running Windows AND Python 2 ?
We can add something like: `@unittest.skipIf(sys.version_info[0] < 2 and platform.system() == 'Windows')` plus required imports (sys, platform)

Reply:
Sure, that works too :)

I'll make those changes.